http/Request.cc

```diff
@@ -0,0 +1,57 @@
+// Class header
+#include "http/Request.hpp"
+
+// Third-party headers
+#include "boost/regex.hpp"
+
+namespace lsst {
+namespace qserv {
+namespace http {
+
+Request::Request(std::shared_ptr<boost::asio::ip::tcp::socket> socket)
+:
+    content(&_requestbuf),
```

Comment:
I'm not sure about the curly inits yet -- see modern effective c++ article for the pros/cons?  That said, if we pick a style for the team and apply consistently I'm okay with that either way.  Let's discuss at the meeting tomorrow and I'm happy to convert this code if the team concensus is pre-curly-init.